src/redux/books/boooks-redux.js

```diff
@@ -0,0 +1,28 @@
+const ADD_BOOK = 'bookstore/src/redux/books/addBook';
+const REMOVE_BOOK = 'bookstore/src/redux/books/removeBook';
+
+// action to add
+const addBook = () => ({
+  type: ADD_BOOK,
+  payload: test,
+});
+
+// action to delete
+const removeBook = () => ({
+  type: REMOVE_BOOK,
+  payload: test,
+});
+
+// create a reducer
+const initialState = (state = '', action) => {
+  if (action.type === ADD_BOOK) {
+    return action.payload;
+  }
+  if (action.type === REMOVE_BOOK) {
+    return action.payload;
```

Comment:
- [ ] As required, in the case of where `action.type` is `REMOVE_BOOK`, the function should ES6 filter() method to create a new array, which will not contain the book you want to remove from the store

## EXAMPLE
```
    case REMOVE_BOOK:
      return state.filter((item) => item.item_id !== action.payload.item_id);
```